tests/unit/nupic/research/inhibition_object_test.py

```diff
@@ -39,10 +39,11 @@
 
 class InhibitionObjectTest(unittest.TestCase):
 
+  @unittest.skip("Currently fails due to switch from FDRCSpatial2 to SpatialPooler.")
```

Comment:
It is not possible use the new SP to perform this test due to it not have explicit methods to get inhibition. So I skipped it. Now can remove it, try find alternatives, or then keep it as `skipped` until future review as addressed in https://github.com/numenta/nupic/issues/1895 .